src/snowflake/snowpark/dataframe_analytics_functions.py

```diff
@@ -32,8 +38,8 @@ class DataFrameAnalyticsFunctions:
     To access an object of this class, use :attr:`DataFrame.analytics`.
     """
 
-    def __init__(self, df: "snowflake.snowpark.DataFrame") -> None:
-        self._df = df
+    def __init__(self, dataframe: "snowflake.snowpark.DataFrame") -> None:
```

Comment:
any particular reason why this change was made? I just want to ensure there is no `self._df` the seeps in after we merge and goes undetected

Reply:
yes, the API here is similar to `dataframe_analytics_functions.py`, and `dataframe_stat_functions.py`. However, while they all follow the same implementation pattern, the naming differs for no reason. This here is a fix to get the naming more coherent. (It will allow to simplify the `@publicapi` decorator e.g. when introducing a shared trait). `dataframe` is more descriptive than `df`, that's why the change was made like this here.

For detection, I think we prob. can rely on tests. Any `_df` access should get tested, if not we have a bigger issue I feel...